### PR TITLE
Added sanity test

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

**Review by spangenberg — APPROVED** [review]
LGTM

File: .github/workflows/cq_fetch.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        id: cache-cq-binary
+        uses: actions/cache@v2
+        with:
+          path: cloudquery
+          key: ${{ runner.os }}-${{ hashFiles('cloudquery') }}
+
+      - name: Download Cloudquery
+        if: steps.cache-cq-binary.cache-hit != 'true'
+        run: |
+          curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cloudquery_${OS}_x86_64 -o cloudquery
```
Comment (bbernays) [inline .github/workflows/cq_fetch.yml:53]: I am not sure we can use `latest` here as we bump versions separately from releases

File: test/policy_cq_config.hcl [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Configuration AutoGenerated by CloudQuery CLI
+cloudquery {
+
+  provider "aws" {
+    source = "cloudquery/cq-provider-aws"
+    version = "latest"
```
Comment (bbernays) [inline test/policy_cq_config.hcl:6]: Also this needs to be pegged to a specific version

**Review by bbernays — APPROVED** [review]
LGTM. We will improve this in the future

**Pull request merged**